client/src/components/Connexions/dataWrappers/ConnexionWrapper.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import {compose} from 'redux'
+import {connect} from 'react-redux'
+import {withDataProviders} from 'data-provider'
+
+import {connexionDetailProvider} from '../../../dataProviders/connexionsDataProviders'
+
+const ConnexionWrapper = (WrappedComponent) => {
+  const wrapped = (props) => <WrappedComponent {...props} connexions={props.connexions} />
```

Comment:
doesn't this override the connexions you connect ?

Reply:
:man_facepalming: Yeah, doing connect's work...